Tests/Editor/MockLogger.cs

```diff
@@ -2,23 +2,26 @@
 // SPDX-License-Identifier: Apache-2.0
 
 using System;
-using AmazonGameLiftPlugin.Core.Shared;
 using UnityEngine;
+using AmazonGameLiftPlugin.Core.Shared;
+using ILogger = AmazonGameLift.Editor.ILogger;
 
-namespace AmazonGameLift.Editor
+namespace AmazonGameLiftPlugin.Editor.UnitTests
 {
-    internal sealed class MockLogger : ILogger
+    public class MockLogger : ILogger
```

Comment:
And in case there's any confusion by what I just said. I expect this change is not relevant to this PR and should've been removed with the other namespace changes. I don't see any changes in this PR that would push for this to be `public` all of a sudden

Reply:
Reverted and won't return in another PR. Was an old change that slipped through.